runtime/src/lib.rs

```diff
@@ -110,7 +106,7 @@ impl system::Trait for Runtime {
     type Hash = Hash;
     type Hashing = BlakeTwo256;
     type AccountId = AccountId;
-    type Lookup = Indices;
+    type Lookup = IdentityLookup<AccountId>;
```

Comment:
With this change, as identified by @shamil-gadelshin we were required to add `"Address": "AccountId","LookupSource": "AccountId"` to the types registration in polkadot-js api, which isn't done automatically by the api library unless it detects the `node-template` specName in the runtime.